gnd/src/main/java/com/google/android/gnd/system/StorageManager.java

```diff
@@ -82,7 +78,7 @@ private Completable sendPhotoPickerIntent() {
                   Intent intent = new Intent(Intent.ACTION_GET_CONTENT);
                   intent.setType("image/*");
                   activity.startActivityForResult(intent, PICK_PHOTO_REQUEST_CODE);
-                  Log.d(TAG, "file picker intent sent");
+                  Timber.d("file picker intent sent");
```

Comment:
This method is UI-specific so should probably be defined in the UI layer (e.g., the Fragment) or related helper.

Reply:
Added a TODO for future refactoring